src/overview/components/AddToCart.jsx

```diff
@@ -29,10 +25,18 @@ function AddToCart({ current }) {
       }
     });
   }, [size, currentSizeQty]);
+
+  const handleClick = (e) => {
```

Comment:
Please remove function handleClick on line 29 if it is not being used

Reply:
Done!